nx-X11/programs/Xserver/Xext/Imakefile

```diff
@@ -79,7 +79,7 @@ XF86BIGFOBJS = xf86bigfont.o
 
        SRCS = shape.c $(SHMSRCS) xcmisc.c\
               xtest.c xtest1di.c xtest1dd.c sleepuntil.c \
-	      bigreq.c sync.c $(SCRNSAVSRC) \
+	      bigreq.c sync.c syncsrv.h $(SCRNSAVSRC) \
```

Comment:
On  Mo 18 Jul 2016 15:50:23 CEST, Ulrich Sibiller wrote:

>> @@ -79,7 +79,7 @@ XF86BIGFOBJS = xf86bigfont.o
>>
>>         SRCS = shape.c $(SHMSRCS) xcmisc.c\
>>                xtest.c xtest1di.c xtest1dd.c sleepuntil.c \
>> -	      bigreq.c sync.c $(SCRNSAVSRC) \
>> +	      bigreq.c sync.c syncsrv.h $(SCRNSAVSRC) \
>
> I have been unsure about this, but some lines above I found this,  
> which I took as an example:
> ```
> XF86BIGFSRCS = xf86bigfont.c xf86bigfontsrv.h
> ```

Ah, ok. I think that it was not a good example...

;-)
-- 

DAS-NETZWERKTEAM
mike gabriel, herweg 7, 24357 fleckeby
mobile: +49 (1520) 1976 148
landline: +49 (4354) 8390 139

GnuPG Fingerprint: 9BFB AEE8 6C0A A5FF BF22  0782 9AF4 6B30 2577 1B31
mail: mike.gabriel@das-netzwerkteam.de, http://das-netzwerkteam.de